app/controllers/documents_controller.rb

```diff
@@ -23,6 +23,10 @@ def show
     @edition = Edition.find_current(document: params[:document])
   end
 
+  def history
+    @edition = Edition.find_current(document: params[:document])
```

Comment:
I suspect here that we're also going to want to pull out timeline entries rather have that done in the view (it'll be essential for pagination as well as just a nice separation of concerns improvement)

https://github.com/alphagov/content-publisher/blob/bb263e3b8438f074f17a3f49e486405fdf16f108/app/views/documents/show/_history_tab.html.erb#L5